.devcontainer/devcontainer.json

```diff
@@ -33,26 +33,19 @@
         "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
         "python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
         "python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-        "[python]": {
-          "editor.defaultFormatter": "ms-python.black-formatter"
-        }
+        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint"
       },
 
       // Add the IDs of extensions you want installed when the container is created.
-      "extensions": [
-        "ms-python.python",
-        "ms-python.vscode-pylance",
-        "ms-python.black-formatter"
-      ]
+      "extensions": ["ms-python.python", "ms-python.vscode-pylance","batisteo.vscode-django"]
```

Comment:
Suggestion: I use a coverage extension to quickly check if new tests cover all new code @luwol03 @SchrodingersGat would that be a reasonable addition? We would need to add `coverage.py` to the dev-reqs then probably.

Reply:
Do you know how to improve the testing setup as its pretty bad marked in https://github.com/inventree/InvenTree/issues/3384#issuecomment-1206799712 ? I tried to setup the test extension, but didn't got it working, so I sticked with the CLI and prints.

Reply:
@luwol03 we are using the default solution for Django. It is the same as in most projects.
The test extension normally auto-discovers test if you chose the right framework at the start (unittest or pytest).

I really do not care how the code gets developed as long as the prints do not make into production code. My suggestion would be to use the debug console or start the Django shell and develop the tests in there. Most test consist of short one-liners so that usually works fine.

Reply:
Which is the correct one? unites or pytest?

Reply:
@luwol03 they support the same call API. While we use unittest, both will work.